docs/reach/rs2/led-status.md

```diff
@@ -1,48 +1,60 @@
-| <div style="text-align: center;"> **Base statuses** </div> |
-| <br><br><br><br><div style="text-align: center;">    Correction output ON   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/base-mode.png" style="height: 150px;"></div>   |
-| <br><br><br><br><div style="text-align: center;">    Both Correction Input and Correction output ON, <br>  the device holds fix   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/correction-in-out.gif" style="height: 150px;"></div>   |
+![](../img/reachrs2/led-status/rs2.png){.reciever}
+
+<div class="reciever-wrapper">    
+<div class="reciever-container">
+<div class="reciever-description">
+    <h4>Battery</h4>
+    <p>The lights above the power button indicate the battery charge level. If the loading animation is over, then you can connect your phone to Reach.</p>
+</div>
+<div class="reciever-status">
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/booting.gif"><span>Loading</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/full-charge.jpg"><span>Full charge</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/charging.gif"><span>Charging</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/low-battery.jpg"><span>Low battery</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/Nech_boot.gif"><span>Not enough charge to boot</span></div>
+</div>
+</div>
+<div class="reciever-container">
+    <div class="reciever-description"> 
+    <h4>Network</h4>
+    <p>During boot, Reach RS2 enters a network scan state and searches for known networks to connect to. If it didn’t find a known network, it switches to hotspot mode.</p>
+    </div>
+    <div class="reciever-status">
+        <div class="status-wrapper"><img src="../img/reachrs2/led-status/broadcasting.png"><div class="wrapper-text">
+        <span class="op-45">Solid:</span><span>Broadcasting Wi-Fi</span></div></div>
+        <div class="status-wrapper"><img src="../img/reachrs2/led-status/network-led.png"><div class="wrapper-text">
+        <span class="op-45">Solid:</span><span>Connected to Wi-Fi network</span></div></div>
+        <div class="status-wrapper"><img src="../img/reachrs2/led-status/network scan.gif"><div class="wrapper-text">
+        <span class="op-45">Blinking:</span><span>Scanning networks</span></div></div>
+    </div>
+</div>
+<div class="reciever-container">
+    <div class="reciever-description"><h4>RTK Status</h4>
+    <p>This LED is used to display ReachView RTK status. RTK status has two colors: white and blue. White indicates incoming corrections, and blue indicates whether outgoing corrections are enabled. If the outgoing ones are turned off, there will only be a white diode. If enabled, the light will switch between white and blue every two seconds.</p>
```

Comment:
_diode_ => I'd change it to LED. Generally speaking, LED and diode are not the same